lib/scss/stanford_events.node.scss

```diff
@@ -0,0 +1,133 @@
+@charset 'UTF-8';
+@import "./stanford_events.config";
+
+.section-editorial-content {
+  @include modular-spacing('padding', 6 null 3 null);
+  @include modular-spacing('margin', null null 5 null);
+
+  color: $su-color-black;
+
+  h1 {
+    @include modular-spacing('margin', null null -1 null);
+    @include modular-typography(5);
+  }
+
+  .su-event-subheadline {
+
+  }
+
+  .su-event-dek {
+    @include big-paragraph;
+    @include modular-spacing('margin', null null 4 null);
+  }
+
+  .su-event-sponsor {
+    @include subheading;
+  }
+}
+
+.body-region {
+  .node-stanford-event-su-event-type {
+    @include modular-spacing('margin', 5 null);
+
+    h2 {
+      @include big-paragraph;
+      @include modular-spacing('margin', null null -3 null);
+      @include modular-spacing('padding', 0 null null null);
+
+      border-top: 1px solid $su-color-cool-grey-15;
+      font-weight: $su-font-regular;
+    }
+  }
+}
+
+.right-region {
```

Comment:
This selector doesn't seem specific enough. It is possible that when a view loads an event node as a rendered entity that this stylesheet could be included. Perhaps nest this selector in the layout selector?

Reply:
@jenbreese see above comment.